deploy/osps/default/osp-flatcar.yaml

```diff
@@ -654,6 +654,14 @@ spec:
               [Install]
               WantedBy=multi-user.target
 
+      - path: /etc/systemd/system/kubelet.service.d/extras.conf
+        content:
+          inline:
+            encoding: b64
```

Comment:
Just a small nit; please remove the encoding. This is noop when using `ignition` so it's best if we don't specify this.

Reply:
done